timemachine/fe/chiral_utils.py

```diff
@@ -398,3 +401,72 @@ def enumerate_planar_torsions(mol):
     find_flipped_planar_torsions = partial(_find_flipped_torsions, planar_torsions_a, planar_torsions_b)
 
     return find_flipped_planar_torsions
+
+
+def make_chiral_restr_fxns(mol_a, mol_b, chiral_k: float = DEFAULT_CHIRAL_ATOM_RESTRAINT_K):
+    restr_idxs_a = np.array(setup_all_chiral_atom_restr_idxs(mol_a, get_romol_conf(mol_a)))
+    restr_idxs_b = np.array(setup_all_chiral_atom_restr_idxs(mol_b, get_romol_conf(mol_b)))
+
+    @jit
+    def U_a(x_a):
+        return U_chiral_atom_batch(x_a, restr_idxs_a, chiral_k).sum()
+
+    @jit
+    def U_b(x_b):
+        return U_chiral_atom_batch(x_b, restr_idxs_b, chiral_k).sum()
+
+    return U_a, U_b
+
+
+def xs_ab_from_xs(xs: NDArray, atom_map):
```

Comment:
```suggestion
def xs_ab_from_xs(xs: NDArray, atom_map: AtomMapMixin):
```

Reply:
The reason this isn't imported it is that it creates a circular import. I had tried that originally and then just tried putting in the string but Mypy complained.

Will add a docstring so it is clear

Reply:
Added docstring in https://github.com/proteneer/timemachine/pull/1263/commits/05cca3e74ba7f663f87a9647e4cea37d73bc740b